evcouplings/complex/protocol.py

```diff
@@ -74,6 +123,32 @@ def _modify_segments(seg_list, seg_prefix):
 
     return outcfg
 
+def map_frequencies_file(frequencies_file, outcfg, **kwargs):
+    """
+    Renumbers the frequencies file created in complex concatenation
```

Comment:
maybe explain more clearly why the file needs to be corrected in docstring?

Reply:
Tried to clarify - it's because otherwise the numbering ends up being continuous for the whole concatenated alignment rather than segment-aware